src/main/java/jp/ac/keio/bio/fun/xitosbml/cli/CliMainImgSpatial.java

```diff
@@ -76,4 +112,71 @@ public void runCli(String imagePath, String outputPath) {
 
 	}
 
+	protected void cli(ImagePlus imager) {
+		hashDomainTypes = new HashMap<String, Integer>();
+		hashSampledValue = new HashMap<String, Integer>();
+		// imager.show();
+		trial = new TrialForImg(hashDomainTypes, hashSampledValue, imager);
+		// HashMap<String, ImagePlus> hashDomFile = trial.getDomFile();
+		// System.out.println(hashDomFile.values());
+	}
+
+	/**
+	 * Following process is performed to an image:
+	 * <ol>
+	 * <li>Interpolate an image if it is a Z-stack image (3D image) and the voxel
+	 * size of each axis (x, y and z) is not equal</li>
+	 * <li>Fill holes (blank pixels) in the image by morphology operation if
+	 * exists</li>
+	 * <li>Add a membrane between two different domains if exists</li>
+	 * </ol>
+	 *
+	 * The converted image will be generated as
+	 * {@link jp.ac.keio.bio.fun.xitosbml.image.SpatialImage}, which is a base class
+	 * for representing spatial image in XitoSBML.
+	 */
+
+	protected void computeImgTrial() {
```

Comment:
Why is it called trial?

Reply:
Because I was trying to compute the image at that point in time and forgot to change the name. Should I rename the method computeImg()?